### PR TITLE
Added support for VM NIC rxd txd

### PR DESCRIPTION
**Pull request by ballle98**
Needed more control over receive and transmit descriptors

### CODE REVIEW

File: ovs_performance.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python
```
Comment (apconole) [inline ovs_performance.py]: Any reason why this was removed?  I think it doesn't belong with this commit.

Comment (ballle98) [reply ovs_performance.py]: your right that was a mistake

**Review by ballle98 — COMMENTED** [review]
reverted change on line 1

File: ovs_performance.py (hunk 1 above)
Comment (ballle98) [reply ovs_performance.py]: fixed

**Comment by chaudron** [comment]
Thanks for the contribution!

**Pull request merged**